backend/src/main/java/com/bakdata/conquery/models/datasets/concepts/filters/postalcode/PostalCodeDistanceEntity.java

```diff
@@ -0,0 +1,36 @@
+package com.bakdata.conquery.models.datasets.concepts.filters.postalcode;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+
+import com.univocity.parsers.annotations.Parsed;
+import io.dropwizard.validation.ValidationMethod;
+import lombok.Data;
+import lombok.NonNull;
+
+@Data
+public class PostalCodeDistanceEntity {
```

Comment:
Es ist denke ich besser eine Datenstruktur zu nehmen, die für einen zweiwertigen Lookup gemacht ist: https://www.baeldung.com/guava-table

Reply:
was ist der Vorteil davon ?

Reply:
Dann kannst du einen Teil der Klassen dadurch ersetzen.
Du befüllst die hälfte der Tabelle.
Und sammelst dann die PLZ im Radius ein in dem du dir einmal die RowMap und einmal die ColumnMap einer PLZ ausgibts und aus beiden die rausfilterst die zum Radius passen.

Ist eine Idee, die Performance von Table ist vielleicht für sparse Tables besser.